core/templates/pages/landing-pages/topic-landing-page/topic-landing-page.mainpage.html

```diff
@@ -13,8 +13,10 @@
           </content>
           <page-footer></page-footer>
         </base-content>
+        <page-footer>
+          <oppia-footer></oppia-footer>
+        </page-footer>
```

Comment:
Shouldn't this be inside the `base-content` tag within the page-footer tag in line 14?
@DubeySandeep PTAL

Reply:
Thanks for your comment, @ashutoshc8101! I've created a PR https://github.com/oppia/oppia/pull/12972 to fix the issue.